packages/account-v2/src/modules/IDVForm/IDVForm.tsx

```diff
@@ -17,9 +17,9 @@ type TIDVFormProps = {
 };
 
 type TIDVFormValues = {
-    document_additional?: string;
-    document_number: string;
-    document_type: string;
+    documentAdditional?: string;
```

Comment:
For our own implementation, can we rename `documentAdditional` to `additionalDocument` which sounds more reasonable